index.js

```diff
@@ -94,11 +94,18 @@ function createLoaderRules(languages, features, workers, publicPath) {
   const workerPaths = workers.reduce((acc, { label, output }) => Object.assign(acc, {
     [label]: `${publicPath ? `${stripTrailingSlash(publicPath)}/` : ''}${output}`,
   }), {});
+
+  const getWorkerPath = (workerPath) => {
+    if(!publicPath && window.__webpack_public_path__) {
+      return `${window.__webpack_public_path__}/${workerPath}`
+    }
+    return workerPath;
+  };
+
   const globals = {
-    'MonacoEnvironment': `(function (paths) { return { getWorkerUrl: function (moduleId, label) { return paths[label]; } }; } )(${
-      JSON.stringify(workerPaths, null, 2)
-    })`,
+    'MonacoEnvironment': `((paths) => ({ getWorkerUrl: (moduleId, label) => (${getWorkerPath.toString()})(paths[label]) }))(${JSON.stringify(workerPaths, null, 2)})`
```

Comment:
I don't think this change was intentional from you. I recently changed it to use `function` instead of `=>` in order to get this working in IE11.

Reply:
Oops thanks! I will change it back and re-test on my end.